src/components/RecipeEdit.vue

```diff
@@ -61,6 +61,7 @@
             </form>
         </div>
     </div>
+    <router-link to="/" class="back-to-home">Return to Home</router-link>
```

Comment:
Kun je een eigen component maken van de back to home knop. en bijvoorbeeld globale button styling die over de gehele website kan worden gebruikt zo staan niet in meerdere bestanden dezelfde css code voor hetzelfde type button